src/test/java/com/uid2/admin/vertx/SearchServiceTest.java

```diff
@@ -39,16 +39,23 @@ protected IService createService() {
         return new SearchService(auth, clientKeyProvider, operatorKeyProvider);
     }
 
+    private static Stream<Arguments> searchAsNonMaintainerFailsData() {
```

Comment:
PRIVATE_OPERATOR_SYNC and UNKNOWN I think should also return 401

Reply:
true. I didn't feel necessary to execute all negative test cases and thought 2 invalid roles would suffice